src/app/content/components/Page/scrollToTopOrHashManager.ts

```diff
@@ -26,17 +26,21 @@ const scrollToTarget = (container: HTMLElement | null, hash: string) => {
   }
 };
 
-const scrollToTargetOrTop = (container: HTMLElement | null, hash: string) => {
+const scrollToTargetOrTop = (container: HTMLElement | null, hash: string, previous: boolean) => {
   if (getScrollTarget(container, hash)) {
     scrollToTarget(container, hash);
   } else {
-    scrollToTop();
+    scrollToTop(previous);
   }
 };
 
-const scrollToTop = () => {
+const scrollToTop = (previous: boolean) => {
   const window = assertWindow();
-  resetTabIndex(window.document);
+  if (previous) {
+    document?.querySelector('main')?.focus();
+  } else {
+    resetTabIndex(window.document);
```

Comment:
does this need to happen on initial load? doesn't the focus start at the top by default?

Reply:
You are correct, it works without it.

Reply:
But when I take it out, the lighthouse test fails. Put it back in, it passes.

Reply:
I think the best I can do is inline the resetTabIndex sequence. It seems that the Lighthouse test does not set up with the focus ready to move to the Skip-to link.

Reply:
which test are we talking about?

Reply:
The [a11y Lighthouse](https://github.com/openstax/rex-web/actions/runs/9776275545/job/26988439259?pr=2221#step:6:830) check

Reply:
The [failure](https://github.com/openstax/rex-web/actions/runs/9779824805/job/26999974157#step:6:770) indicates the focus isn't ready to tab to Skip to content.